pandas/io/formats/style.py

```diff
@@ -143,11 +143,16 @@ class Styler(StylerRenderer):
         uses ``pandas.options.styler.format.formatter``.
 
         .. versionadded:: 1.4.0
+    css : dict, optional
+        A dict of strings used to replace the default CSS classes described below.
 
     Attributes
     ----------
     env : Jinja2 jinja2.Environment
-    template : Jinja2 Template
+    template_html : Jinja2 Template
```

Comment:
useful to document these?

Reply:
there is something in the user guide. can do as follow on if you think needs more.